### PR TITLE
feat(radar_object_fusion_to_detected_object): enable confidence compensation in radar fusion

### PR DESCRIPTION
**Pull request by scepter914**
Signed-off-by: scepter914 <scepter914@gmail.com>

## Description

Add confidence compensation in radar fusion.
Along with that, fix bug of confidence hundling and add debug topic for radar fusion package.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by yukkysaito** [comment]
@scepter914 
The title is in radar fusion, but it is enabled for lidar only and camera_lidar as well.
Can you change the PR title?

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 1]
```diff
@@ -27,6 +27,7 @@
   <arg name="use_pointcloud_container" default="false" description="use pointcloud container for detection preprocessor"/>
   <arg name="container_name" default="pointcloud_container"/>
   <arg name="use_validator" default="true" description="use obstacle_pointcloud based validator"/>
+  <arg name="score_threshold" default="0.45"/>
```
Comment (yukkysaito) [inline launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:30]: @yukke42 Can you check this value?

**Comment by scepter914** [comment]
> The title is in radar fusion, but it is enabled for lidar only and camera_lidar as well.

Confidence compensation is just  the feature of radar fusion and is not relate to LiDAR or Camera-LiDAR fusion.
I changed LiDAR or Camera-LiDAR fusion launcher to enable confidence compensation option of radar fusion, but it doesn't affect LiDAR and Camera-LiDAR fusion.

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 1 above)
Comment (scepter914) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:30]: (I think it is default value of CenterPoint now.)
https://github.com/autowarefoundation/autoware.universe/blob/main/perception/lidar_centerpoint/launch/lidar_centerpoint.launch.xml#L8

Comment (yukke42) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:30]: Yes, It's the default value of lidar_centerpoint.

**Review by yukkysaito — APPROVED** [review]
LGTM
@miursh pls double check

**Pull request merged**